src/Libraries/DynamoUnits/Units.csproj

```diff
@@ -21,10 +21,8 @@
     <PackageReference Include="System.Resources.Extensions" Version="5.0.0" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="ForgeUnits.NET" Version="4.0.3" GeneratePathProperty="true">
-        <ExcludeAssets>build</ExcludeAssets>
-    </PackageReference>
-	  <PackageReference Include="ForgeUnits.Schemas" Version="1.0.0" GeneratePathProperty="true" />
+    <PackageReference Include="ForgeUnits.NET" Version="4.0.3" />
```

Comment:
it appears you added this to the net6 section and not the net4.8 one - though from the binary diff it seems that it was not being copied correctly during the net6 build correctly either.

Reply:
I changed the common package. (both target frameworks)

Reply:
ah, thanks @pinzart90 I missed that - though what do you make of the binary diff?